module/VuFind/src/VuFind/Controller/Plugin/AbstractRequestBase.php

```diff
@@ -354,4 +357,14 @@ protected function getDateFromArray($dateArray)
             date('Y') + $y
         );
     }
+
+    /**
+     * Get remembered valid IDs
+     *
+     * @return array
+     */
+    protected function getValidIds(): array
```

Comment:
To @EreMaijala's point, if we keep this public here, we avoid backward compatibility breaks. Would you like to change it back to public so we can merge to dev, and then I can make it protected in dev-9.0, or do you prefer to simply merge this change directly to dev-9.0?

Reply:
@demiankatz You can merge this change only to dev-9.0.